validator/registration.py

```diff
@@ -53,14 +53,15 @@ async def fetch_registered_agents(self) -> None:
                     for agent in active_agents
                 }
                 logger.info("Successfully fetched and updated active agents.")
+                await self.update_agents_profiles_and_emissions()
```

Comment:
do we need this here?  this is a separate thread that has it's own cadence / loop.  agent registration check happens every 1 minute, whereas the updater is happening every hour - this makes it also happen every minute